modules/developer_manual/pages/app/fundamentals/publishing.adoc

```diff
@@ -310,7 +311,17 @@ code. This does not depend on whether it happens intentionally or not.
 | Max. 4000 characters; Provide all necessary, detailed information about the product. This should contain all user relevant information. Don’t get lost in technical details, focus on the benefits the product offers; Also, use markdown to layout your description.
 
 | license
-| At the moment following license are available: - ocl - ownCloud commercial license; ownCloud enterprise apps only - agpl - mit version The release version of your app. Note: To overwrite a release (using the same version number) it must be in state ``planned``. Once published, you cannot replace a release.
+a| At the moment following license are available:
+
+- https://en.wikipedia.org/wiki/Open_Content_License[OCL]
+- https://owncloud.com/licenses/owncloud-commercial/[ownCloud Commercial License]
++
+NOTE: This is for ownCloud Enterprise Apps *only*
+- https://en.wikipedia.org/wiki/Affero_General_Public_License[AGPL]
+- https://en.wikipedia.org/wiki/MIT_License[MIT License].
+
```

Comment:
maybe you want to add a + here too so that the next note gets written into the same cell but next line

Reply:
It does that already. 
![bildschirmfoto 2018-11-22 um 11 41 46](https://user-images.githubusercontent.com/196801/48897890-a13eb780-ee4b-11e8-8298-7baf75b2664a.png)